src/components/PopUpBar/index.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from "react-redux";
+import PopUpBar from "./PopUpBar";
+
+
+const mapStateToProps = (state) =>({
+    auth: state.auth.user,
+    userProfile: state.userProfile
+})
+
+const mapDispatchToProps=(dispatch)=>({
+
+})
+
```

Comment:
we don't need an empty dispatch to props objects, if no actions we can use undefined for that instead of declaring an empty object like this